### ISSUE TITLE
WebRTC: Chrome HEVC support

### ISSUE THREAD

**Issue by shuwang-focusai** [ossrs/srs#4178]
SRS 6.0 already supports a lot HEVC functions, but not RTMP2WebRTC because Chrome/FireFox don't support HECV in WebRTC.

From current report, Chrome will launch the feature to support HEVC in WebRTC (https://issues.chromium.org/issues/341986313).
Is there any plan for SRS to support RTMP2WebRTC in HEVC? Thanks!

**Comment by winlinvip** [comment]
Patch welcome.

**Comment by duiniuluantanqin** [comment]
The pull request https://github.com/ossrs/srs/pull/4289 introduces support for HEVC over WebRTC. Please wait patiently for it to be merged.

**Comment by zapf2000** [comment]
It should work if you start Chrome like this: chrome.exe --enable-features=PlatformHEVCEncoderSupport,WebRtcAllowH265Receive,WebRtcAllowH265Send --force-fieldtrials=WebRTC-Video-H26xPacketBuffer/Enabled

It's not enabled by default.

**Comment by duiniuluantanqin** [comment]
> It should work if you start Chrome like this: chrome.exe --enable-features=PlatformHEVCEncoderSupport,WebRtcAllowH265Receive,WebRtcAllowH265Send --force-fieldtrials=WebRTC-Video-H26xPacketBuffer/Enabled
> 
> It's not enabled by default.


Yes, for additional details, please refer to: https://github.com/ossrs/srs/pull/4289#issue-2852838971

`TRANS_BY_GPT4`
